kernels/volk/volk_4ic_deinterleave_8i_x2.h

```diff
@@ -0,0 +1,109 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Derek Kozel
+ *
```

Comment:
This license header needs adjustment.

Reply:
Updated to the standard Copyright 2020 FSF.. Part of GNU Radio.